src/redux/api/simpleMode.js

```diff
@@ -0,0 +1,27 @@
+import { backendApi } from './backendApi';
+import moment from 'moment/moment';
+import { Requests } from '../../helpers/Api';
+
+export const wholeSimpleMode = ['SimpleMode'];
+
+export const simpleModeApi = backendApi.injectEndpoints({
```

Comment:
Did you move only simple mode to RTK2? Or everything?

Reply:
Only simple mode. Step-by-step.

Reply:
V